INTHEWILD.md

```diff
@@ -290,6 +290,7 @@ Currently, **officially** using Airflow:
 1. [MTsolutions](https://www.mtsolutions.io)[[@slara](https://github.com/slara)]
 1. [Multiply](https://www.multiply.com) [[@nrhvyc](https://github.com/nrhvyc)]
 1. [Mutt Data](https://muttdata.ai/) [[@plorenzatto](https://github.com/plorenzatto)]
+1. [Narrativa](https://www.narrativa.com/)
```

Comment:
You'd have to move it down - it's not alphabetically sorted and static check will fail